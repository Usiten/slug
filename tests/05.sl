= if (10 < 3) {
	16
};

a = if (1 < 2) {
	0; 1; 2; 255
};

= if (1) {
	if (2) {
		300
	}; 
	42;
	if (a) {
		0
	}
};