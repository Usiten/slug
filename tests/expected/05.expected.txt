-- Stack Top --
>0
 42
 300
 2
 1
 0
---------------
-- Var Data --
a: 255
---------------
